flake.lock

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Is this file supposed to be committed?

Reply:
yes, commited on purpose so that the same dependency versions' are used